documentation/docs/welcome.md

```diff
@@ -15,29 +15,29 @@ keywords:
 
 # Welcome to INX-Dashboard
 
-INX-Dashboard is a node configuration dashboard for node owners.
-It provides an overview of your nodes health and allows you to manage peers and plugins.
-You can browse the nodes database with the built-in explorer and check the current tangle activity in the visualizer.
+INX-Dashboard provides a web GUI tool to help you manage your node. It provides an overview of the node's status and health and allows you to choose its peers, as well as the enabled plugins. It has a built-in explorer of all nodes on the network and visualizer for activity on the Tangle.
 
 ## Setup
 
-The recommended setup is to use the provided [Docker images](https://hub.docker.com/r/iotaledger/inx-dashboard).
-These images are also used in our [HORNET recommended setup using Docker](http://wiki.iota.org/hornet/develop/how_tos/using_docker).
+We recommend you to use the [Docker images](https://hub.docker.com/r/iotaledger/inx-dashboard).
+These images are also used in the [Docker setup](http://wiki.iota.org/hornet/develop/how_tos/using_docker) of Hornet.
```

Comment:
```suggestion
The Hornet smart contract node also uses these images for the [Docker setup](http://wiki.iota.org/hornet/develop/how_tos/using_docker).
```

Reply:
I think here it works better in passive voice.